resolwe_bio/processes/workflows/mirna.yml

```diff
@@ -3,66 +3,335 @@
+                Default: 5
       # this set of parameters is for featureCounts
-    - name: annotation
-      label: Annotation (GTF/GFF3)
-      type: data:annotation
-    - name: id_attribute
-      label: ID attribute
-      type: basic:string
-      default: gene_id
-      allow_custom_choice: true
-      choices:
-        - label: gene_id
-          value: gene_id
-        - label: transcript_id
-          value: transcript_id
-        - label: ID
-          value: ID
-        - label: geneid
-          value: geneid
-      description: |
-        GTF/GFF3 attribute to be used as feature ID. Several GTF/GFF3 lines
-        with the same feature ID will be considered as parts of the same
-        feature. The feature ID is used to identify the counts in the
-        output table. In GTF files this is usually 'gene_id', in GFF3 files
-        this is often 'ID', and 'transcript_id' is frequently a valid
-        choice for both annotation formats.
-    # this next option could perhaps be hidden and hard-coded into the workflow
-    - name: feature_class
-      label: Feature class
-      type: basic:string
-      default: miRNA
-      description: |
-        Feature class (3rd column in GFF file) to be used, all features of other
-        types are ignored.
+    - name: quant_options
+      label: Quantification options
+      group:
+        - name: annotation
+          label: Annotation (GTF/GFF3)
+          type: data:annotation
+        - name: id_attribute
+          label: ID attribute
+          type: basic:string
+          default: Name
+          allow_custom_choice: true
+          choices:
+            - label: miRNA name
+              value: Name
+            - label: gene_id
+              value: gene_id
+            - label: transcript_id
+              value: transcript_id
+            - label: ID
+              value: ID
+            - label: geneid
+              value: geneid
+          description: |
+            GTF/GFF3 attribute to be used as feature ID. Several GTF/GFF3 lines
+            with the same feature ID will be considered as parts of the same
+            feature. The feature ID is used to identify the counts in the
+            output table. In GTF files this is usually 'gene_id', in GFF3 files
+            this is often 'ID', and 'transcript_id' is frequently a valid
+            choice for both annotation formats. miRNA name refers to the miRBase
+            GFF3 'Name' filed and is the default option.
+        - name: feature_class
+          label: Feature class
+          type: basic:string
+          default: miRNA
+          description: |
+            Feature class (3rd column in GFF file) to be used, all features of other
+            types are ignored. Default: miRNA.
+        - name: assay_type
+          label: Assay type
+          type: basic:string
+          default: forward
```

Comment:
Is the `assay_type` set in accordance with a specific kit requirement? It's not wise to set the default value for this input and hide it under advanced section. If data from multiple kits with different strand-specificity are to be used with this pipeline, then this can lead to unintended errors. It's best to remove the default value, move this input out of the advanced section, and set the appropriate kit-relatated default elsewhere.